Bang/GameEvents/CardEffects/GatlingActionHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Bang.Characters;
+using Bang.Characters.Visitors;
+using Bang.Game;
+using Bang.GameEvents.CardEffects.States;
+using Bang.Players;
+using Bang.PlayingCards;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bang.GameEvents.CardEffects
+{
+    internal class GatlingActionHandler : CardActionHandler
+    {
+        public override HandlerState ApplyEffect(Game.Gameplay gameplay, Player attackPlayer, BangGameCard card)
+        {
+            var victimStatesList = new Dictionary<Player, HandlerState>();
+
+            foreach (var victim in gameplay.Players.Where(p => p.Id != attackPlayer.Id))
```

Comment:
I think it will be better to apply gatling card effect to other alive players.

Reply:
Nice catch, thanks